jewish/date.py

```diff
@@ -294,6 +296,14 @@ def english_month_name(self):
         else:
             return names[self.month - 1]
 
+    def hebrew_month_name(self):
+        names = ('תשרי', 'חשוון', 'כסלו', 'טבת', 'שבט', 'אדר א\'',
```

Comment:
The tuple is not viewed correctly in here (at least with firefox). You can view it with a editor that isn't trying to be smart (I used vim).